common/lib/xmodule/xmodule/js/fixtures/problem_content.html

```diff
@@ -12,11 +12,13 @@ <h3 class="hd hd-2 problem-header">Problem Header</h3>
     <span id="display_example_1"></span>
     <span id="input_example_1_dynamath"></span>
 
-    <button class="check Check" data-checking="Checking..." data-value="Check"><span class="check-label">Check</span><span class="sr"> your answer</span></button>
+    <button class="check Submit" data-checking="Submitting..." data-value="Submit"><span class="check-label">Submit</span><span class="sr"> your answer</span></button>
```

Comment:
Nit: Should the class be capitalized?  I am still learning the style guidelines

Do we need translations on this text?